lib/urldata.h

```diff
@@ -1580,6 +1583,17 @@ enum dupstring {
   STRING_LAST /* not used, just an end-of-list marker */
 };
 
+enum dupblob {
+  BLOB_CERT_ORIG,
+  BLOB_CERT_PROXY,
+  BLOB_KEY_ORIG,
+  BLOB_KEY_PROXY,
+  BLOB_SSL_ISSUERCERT_ORIG,
+  BLOB_SSL_ISSUERCERT_PROXY,
```

Comment:
You're adding more blobs to this table than what exists. I think we should rather add them if/when we actually get support for those.

Reply:
I removed BLOB_SSL_ISSUERCERT_PROXY and support all three other.
If seem STRNIG_SSL_ISSUERCERT_PROXY is not fully supported (I 'm not sure)